src/Uno.UWP/Devices/Geolocation/Geolocator.iOSmacOS.cs

```diff
@@ -21,10 +21,6 @@ public Geolocator()
 				DesiredAccuracy = DesiredAccuracy == PositionAccuracy.Default ? 10 : 1,
 			};
 
-#if __IOS__ //required only for iOS
-			_locationManager.RequestWhenInUseAuthorization();

```

Comment:
You don't request it somewhere else?

Reply:
yes! it's already being asked in RequestAccessAsync, so we don't need to add it there.